src/features/content-explorer/messages.js

```diff
@@ -118,6 +118,31 @@ const messages = defineMessages({
         description: "Results label for number of items on list when it's just 1",
         id: 'boxui.contentExplorer.result',
     },
+    includeSubfolders: {
+        defaultMessage: 'Include Subfolders',
+        description: 'Label text shown next to the Include Subfolders toggle',
+        id: 'boxui.contentExplorer.includeSubfolders',
+    },
+    includeSubfoldersDefaultInformation: {
+        defaultMessage: 'When enabled, report is run on a single selected folder and all its subfolders.',
```

Comment:
this message is specifying `report` which is specific to reports. could we generalize this? maybe we can pass the tooltip component or message as a prop?

Reply:
Pass it all the way down from the parent component?